ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -541,7 +541,12 @@ private void respondToPing(
     if (packetData.getExpiration() < Instant.now().getEpochSecond()) {
       return;
     }
-    final PongPacketData data = PongPacketData.create(packetData.getFrom(), pingHash);
+    final PongPacketData data =
+        PongPacketData.create(
+            sender
+                .getEndpoint() /* We don't care about the `from` field of the ping, we pong to the `sender` */,
```

Comment:
Would this comment look better before?  So it formats w/o breaking up the `sender.getEndPoint()`